README.md

```diff
@@ -97,20 +113,22 @@ Let's take an example:
     ```adblock
     ! Title: List 2
     ! Diff-Path: ../patches/batch.patch
+    ! Diff-Name: list2
     ```
 
-  `Diff-Path` is relative to `list2.txt` location so the final URL of the diff file will be `https://example.com/patches/batch.patch`.
+  * `Diff-Path` is relative to `list2.txt` location so the final URL of the diff file will be `https://example.com/patches/batch.patch`.
+  * `Diff-Name` is mandatory for lists that support batch differential updates.
 
 * `batch.patch`
 
-  A file that contains patches for both `list1.txt` and `list2.txt`. The `f` directive allows applying this patch to both `list1.txt` and `list2.txt`. Note, that the file path in the `f` directive is relative to the patch file location.
+  A file that contains patches for both `list1.txt` and `list2.txt`. It uses the `diff name:` directive to point at which patch should be applied to which list.
 
     ```diff
-    f ../list1/list1.txt [checksum] 3
+    diff name:list1 checksum:atatata lines:3
```

Comment:
Format of checksum is hex string, so example given should contain only hex characters to be sure there is no misunderstanding.